src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+package main.java;
+
+public class Event extends Task{
+    protected String date;
+    public Event(String n, boolean d, String dat){
+        super(n, d);
+        date = dat;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]"+super.toString()+" (at: "+date+")";
```

Comment:
Same as other file, consider adding spaces between operands and binary operators